### ISSUE TITLE
fontTools 4.0 breaks DrawBot module

### ISSUE THREAD

**Issue by frankrolf** [typemytype/drawbot#290]
You’re probably aware of this, but just in case:

`import drawBot`

```
Traceback (most recent call last):
  File "db_test.py", line 1, in <module>
    import drawBot
  File "/usr/local/lib/python3.7/site-packages/drawBot-3.118-py3.7.egg/drawBot/__init__.py", line 3, in <module>
    from .drawBotDrawingTools import _drawBotDrawingTool
  File "/usr/local/lib/python3.7/site-packages/drawBot-3.118-py3.7.egg/drawBot/drawBotDrawingTools.py", line 11, in <module>
    from .context import getContextForFileExt, getContextOptions, getFileExtensions, getContextOptionsDocs
  File "/usr/local/lib/python3.7/site-packages/drawBot-3.118-py3.7.egg/drawBot/context/__init__.py", line 3, in <module>
    from .pdfContext import PDFContext
  File "/usr/local/lib/python3.7/site-packages/drawBot-3.118-py3.7.egg/drawBot/context/pdfContext.py", line 11, in <module>
    from .baseContext import BaseContext
  File "/usr/local/lib/python3.7/site-packages/drawBot-3.118-py3.7.egg/drawBot/context/baseContext.py", line 11, in <module>
    from fontTools.misc.py23 import basestring, PY2, unichr
ImportError: cannot import name 'PY2' from 'fontTools.misc.py23' (/usr/local/lib/python3.7/site-packages/fontTools/misc/py23.py)```

**Comment by justvanrossum** [comment]
Duplicate of #288, which will be fixed by #289.

**Comment by frankrolf** [comment]
Great, thanks!
